src/cmap/connect.ts

```diff
@@ -125,6 +127,25 @@ function performInitialHandshake(
         return;
       }
 
+      if (options.loadBalanced) {
+        // TODO: Durran: Remove when server support exists.
```

Comment:
Let's make a ticket from this TODO

Reply:
https://jira.mongodb.org/browse/NODE-3431

Reply:
Thanks! Lets put the ticket in the comment to keep track

Reply:
Added ticket to comment.